drivers/pinctrl/Kconfig.imx

```diff
@@ -26,6 +26,12 @@ config PINCTRL_IMX_SCMI
 # TODO: Find better place for this option
 config MCUX_XBARA
 	bool "MCUX XBARA driver"
-	depends on HAS_MCUX_XBARA
+	depends on DT_HAS_NXP_MCUX_XBAR_ENABLED
```

Comment:
These symbols that depend on DT enabled nodes, typically have `default y`, why not here?